From cf8ebbc48491cf5178e4edc57f49ceded20ead55 Mon Sep 17 00:00:00 2001
From: Michael Tokarev <mjt@tls.msk.ru>
Date: Sat, 2 Jun 2012 13:33:42 +0000
Subject: [PATCH spice 4/6] link libspice server with libm libpthread

server/Makefile apparently forgot to link libspice-server
with -lm -lpthread, but it uses symbols from these libraries
directly.  These libs are detected by configure and stored in
$(SPICE_NONPKGCONFIG_LIBS) make variable, but this variable
is never referenced at link time.  Add it to server/Makefile.am,
to libspice_server_la_LIBADD variable.

Signed-off-By: Michael Tokarev <mjt@tls.msk.ru>
---
 server/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/server/Makefile.am b/server/Makefile.am
index b3fcd2c..8b380fc 100644
--- a/server/Makefile.am
+++ b/server/Makefile.am
@@ -40,6 +40,7 @@ libspice_server_la_LIBADD =						\
 	$(SLIRP_LIBS)							\
 	$(SSL_LIBS)							\
 	$(Z_LIBS)							\
+	$(SPICE_NONPKGCONFIG_LIBS)					\
 	$(NULL)
 
 libspice_server_la_SOURCES =			\
-- 
1.8.1

